src/main/java/org/jabref/gui/SaveOrderConfigDisplay.java

```diff
@@ -38,56 +46,87 @@ private void init() {
         fieldNames.add(BibEntry.KEY_FIELD);
         Collections.sort(fieldNames);
         String[] allPlusKey = fieldNames.toArray(new String[fieldNames.size()]);
-        savePriSort = new JComboBox<>(allPlusKey);
+        savePriSort = new ComboBox<>(FXCollections.observableArrayList(allPlusKey));
```

Comment:
You no longer need to use an array. Instead, directly use the `fieldNames` collection.